README.md

```diff
@@ -51,380 +69,30 @@ Different branches of the repo can have different pinned versions of these third
 update submodules after switching branches or pulling recent changes!
 
 To configure git to automatically update submodules when switching branches, run
+
 ```bash
 git config submodule.recurse true
 ```
+
 **NOTE**: this setting will not download **new** or remove **old** submodules with the branch's changes.
 You will have to run the full `git submodule update --init --recursive` command in these situations.
 
-## First Time Setup
```

Comment:
Can you add a link to the quick start docs from the readme? Something along the lines of, "examples of how to train models and develop may be found here".

Reply:
Which page were you thinking? I have something similar to that on line 35:
> Full documentation on using the BioNeMo Framework is provided in our documentation:
<https://docs.nvidia.com/bionemo-framework/latest/user-guide/>.

Reply:
Added a link to the tutorials pages